packages/autocomplete-core/src/types/HighlightHitParams.ts

```diff
@@ -0,0 +1,18 @@
+export type HighlightHitParams<THit> = {
```

Comment:
`HighlightHitParams` is not related to `autocomplete-core`. In your changes, it should be in the example.

Reply:
@gfortaine This comment hasn't been addressed yet. This should be moved to the example, because it's not the responsibility of `autocomplete-core`.

Reply:
+1 `autocomplete-core` is meant to be headless, it doesn't provide components.

For now I'd keep this in the example, likely in `examples/react-renderer/src/Highlight.tsx`.